src/app/clusters/administrator-commissioning-server/AdministratorCommissioningLogic.h

```diff
@@ -0,0 +1,59 @@
+/*
+ *    Copyright (c) 2021-2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/data-model-provider/ActionReturnStatus.h>
+#include <app/server/Server.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+class AdministratorCommissioningLogic
+{
+public:
+    AdministratorCommissioning::CommissioningWindowStatusEnum GetWindowStatus()
+    {
+        return Server::GetInstance().GetCommissioningWindowManager().CommissioningWindowStatusForCluster();
+    }
+
+    const app::DataModel::Nullable<FabricIndex> & GetOpenerFabricIndex()
```

Comment:
So why does this not match the name of the attribute, while GetAdminVendorId does match?